### ISSUE TITLE
Validation metric difference between saving state_dict() and the whole model.

### ISSUE THREAD

**Issue by Grace625** [DingXiaoH/RepLKNet-pytorch#51]
Thanks for your excellent work!

I use RepLKNet as the backbone of my depth estimation network.  After validating the model at training time, I save it and immediately load it to validate again, but I get different validation metrics from the training time. 

I use the standard way in PyTorch to save the state_dict() of RepLKNet, and when I use torch.save() to save the whole model rather than only save the state_dict() of the backbone, this problem disappears. Why does this happen? Looking forward to your reply.
